MQTT_Client_API_Interface.py

```diff
@@ -179,7 +179,10 @@ def on_message(mqtt_client, userdata, msg):
         handle_faulty_reading(reading, "Check device ID", msg.topic)
         return
 
-    forward_reading(reading)
+    try:
+        forward_reading(reading)
+    except BackEndSystemError:
+        pass
```

Comment:
Not thrilled about this, but I'm prioritizing cleaning up the forwarding queued readings function. Forwarding readings here in the callback is only temporary and will be removed when support for request concurrency is added. Adding this expensive call here in the callback is also causing issues with QoS 2 messages. It appears to prevents the client from handing the two way ack with the broker which eventually results in the client locking up during high during high load (broker has queued QoS 1+ messages). QoS 0 and 1 have no issues.